src/Limenius/Liform/Serializer/Normalizer/FormErrorNormalizer.php

```diff
@@ -15,6 +15,7 @@
 use Symfony\Component\Form\FormInterface;
 use Symfony\Component\Serializer\Normalizer\NormalizerInterface;
 use Symfony\Contracts\Translation\TranslatorInterface;
+use Symfony\Contracts\Translation\TranslatorInterface as TranslatorContract;
```

Comment:
Why are you leaving TranslatorInterface along with TranslatorContract? Why not using only one of the two?

Reply:
For backward compatibility with the older versions of Symfony.